.travis.yml

```diff
@@ -17,33 +17,28 @@ notifications:
   email: true
 node_js:
   - '12'
+before_install:
+  - export TZ=Asia/Tokyo
```

Comment:
Temporal fix for RDateRange tests

Reply:
Is this affecting our unit tests? Or is it related with anything else?

Reply:
Only unit tests - I think our timezone mock require updates